drivers/sensor/bmi088/bmi088.c

```diff
@@ -0,0 +1,313 @@
+/* Bosch BMI088 inertial measurement unit driver
+ *
+ * Copyright (c) 2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Datasheet:
+ * https://www.bosch-sensortec.com/media/boschsensortec/downloads/datasheets/bst-bmi088-ds001.pdf
+ */
+
+#define DT_DRV_COMPAT bosch_bmi088
+
+#include <init.h>
+#include <drivers/sensor.h>
+#include <sys/byteorder.h>
+#include <kernel.h>
+#include <sys/__assert.h>
+#include <logging/log.h>
+#include <devicetree.h>
+
+#include "bmi088.h"
+
+#define M_PI   3.14159265358979323846264338327950288
+
+LOG_MODULE_REGISTER(BMI088, LOG_LEVEL_DBG);
```

Comment:
Das sollte am besten mit Kconfig einstellbar sein, siehe andere treiber (bitte testen, das hat bei meinem SPI zeug irgendwie mal nicht funktioniert wie die das gemacht haben (LOG_LEVEL definieren bevor mal `logging/log.h` inkludiert oder so))

Reply:
Ich habe das mal angepasst so wie ich es in anderen drivern gesehen habe. Ich habe aber noch nicht ganz durchblickt worin sich diese Art unterscheidet bzw wieso wir das lieber mit Kconfig einstellen. Habe das Projekt mal bauen lassen und keine Fehler bekommen.